src/core/client/admin/routes/Configure/sections/Auth/SSOKeyRotation/RotationOption.tsx

```diff
@@ -0,0 +1,54 @@
+import { Localized } from "@fluent/react/compat";
+import React, { FunctionComponent } from "react";
+
+export enum RotateOptions {
+  NOW = "NOW",
+  IN10SECONDS = "IN10SECONDS",
+  IN1DAY = "IN1DAY",
+  IN1WEEK = "IN1WEEK",
+  IN30DAYS = "IN30DAYS",
+}
+
+interface Props {
+  value: string;
+}
+
+const RotationOption: FunctionComponent<Props> = ({ value }) => {
+  switch (value) {
+    case RotateOptions.NOW: {
+      return <Localized id="configure-auth-sso-rotate-now">Now</Localized>;
+    }
+    case RotateOptions.IN10SECONDS: {
+      return (
+        <Localized id="configure-auth-sso-rotate-10seconds">
+          10 seconds from now
+        </Localized>
+      );
+    }
```

Comment:
The 10 seconds option was added for testing purposes, it wasn't actually supposed to make it to the figma + final product.

Reply:
lol, fair enough!